banzai/main.py

```diff
@@ -155,7 +155,7 @@ def process_directory(pipeline_context, raw_path, image_types=None, log_message=
     if len(log_message) > 0:
         logger.info(log_message, extra_tags={'raw_path': raw_path})
     image_path_list = image_utils.make_image_path_list(raw_path)
-    image_path_list = image_utils.select_images(image_path_list, pipeline_context.FRAME_SELECTION_CRITERIA,
+    image_path_list = image_utils.select_images(image_path_list, pipeline_context,
                                                 image_types=image_types, db_address=pipeline_context.db_address)
```

Comment:
The keyword arguments `image_types` and `db_address` should be removed

Reply:
Done.